python/servo/command_base.py

```diff
@@ -51,6 +51,10 @@ class CommandBase(object):
     def __init__(self, context):
         self.context = context
 
+        def resolverelative(category, key):
+            if self.config[category][key].startswith("./"):
```

Comment:
We also want to handle `~`.

I'll handle `~` replacement first, then do `path.join`.